boards/arm/mps2_an521/board.cmake

```diff
@@ -17,4 +17,9 @@ if (CONFIG_BUILD_WITH_TFM)
   # TF-M (Secure) & Zephyr (Non Secure) image (when running
   # in-tree tests).
   set(QEMU_KERNEL_OPTION "-device;loader,file=${CMAKE_BINARY_DIR}/tfm_merged.hex")
+
+elseif (CONFIG_SOC_MPS2_AN521_CPU1)
+  set(CPU0_BINARY_DIR ${CMAKE_CURRENT_BINARY_DIR}/zephyr/boards/arm/mps2_an521/empty_cpu0-prefix/src/empty_cpu0-build/zephyr)
```

Comment:
This looks something that could easily break in future.

Reply:
I leave this up to you, @tejlmand , to propose a better solution to this :)
For now I used what was already the solution in the rpmsg samples in the tree. 
Would welcome a proposal to improve this.

Reply:
@tejlmand any change you can provide a hint here?